src/registrar/management/commands/load_organization_data.py

```diff
@@ -0,0 +1,251 @@
+from registrar.management.commands.utility.terminal_helper import TerminalColors, TerminalHelper
+from registrar.management.commands.utility.transition_domain_arguments import TransitionDomainArguments
+from registrar.models import TransitionDomain, DomainInformation
+from django.core.paginator import Paginator
+from typing import List
+
+from registrar.utility.errors import LoadOrganizationError, LoadOrganizationErrorCodes
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Send domain invitations once to existing customers."
+
+    def add_arguments(self, parser):
+        """Add command line arguments."""
+
+        parser.add_argument(
+            "migration_json_filename",
+            help=("A JSON file that holds the location and filenames" "of all the data files used for migrations"),
+        )
+
+        parser.add_argument("--sep", default="|", help="Delimiter character")
+
+        parser.add_argument("--debug", action=argparse.BooleanOptionalAction)
+
+        parser.add_argument("--directory", default="migrationdata", help="Desired directory")
+
+        # Serves as a domain_additional_filename override
+        parser.add_argument(
+            "--domain_additional_filename",
+            help="Defines the filename for additional domain data",
+        )
+
+        # Serves as a organization_adhoc_filename override
+        parser.add_argument(
+            "--organization_adhoc_filename",
+            help="Defines the filename for domain type adhocs",
+        )
+
+    def handle(self, migration_json_filename, **options):
+        """Process the objects in TransitionDomain."""
+
+        # === Parse JSON file === #
+        json_filepath = os.path.join(options["directory"], migration_json_filename)
+
+        # If a JSON was provided, use its values instead of defaults.
+        with open(json_filepath, "r") as jsonFile:
+            # load JSON object as a dictionary
+            try:
+                data = json.load(jsonFile)
+
+                skipped_fields = ["domain_additional_filename", "organization_adhoc_filename"]
+                # Iterate over the data from the JSON file. Skip any unused values.
+                for key, value in data.items():
+                    if value is not None or value.strip() != "":
+                        continue
+                    
+                    # If any key in skipped_fields has a value, then
+                    # we override what is specified in the JSON.
+                    if key not in skipped_fields:
+                        options[key] = value
+
+            except Exception as err:
+                logger.error(
+                    f"{TerminalColors.FAIL}"
+                    "There was an error loading "
+                    "the JSON responsible for providing filepaths."
+                    f"{TerminalColors.ENDC}"
+                )
+                raise err
```

Comment:
Most of this is copied over from the last PR. I am unsure if this needs more clarity.